src/components/VideosModal.js

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import PropTypes from "prop-types";
+import {Collapsible, CollapsibleItem, Icon } from 'react-materialize';
```

Comment:
`Icon` can be removed from the imports, as it's not being used. That'll help clear up the console warning.